sharding/contracts/sharding_manager.sol

```diff
@@ -87,14 +87,9 @@ contract SMC {
 
       // Get the most recent block number before the period started
         uint latestBlock = period * PERIOD_LENGTH - 1;
+        uint index = uint(keccak256(block.blockhash(latestBlock), _index, shardId)) % sampleSize;
```

Comment:
Is it possible to make this line a bit simpler? or abstract away with variables?